solana-programs/anchor/audius-data/tests/test-helpers.ts

```diff
@@ -116,14 +116,14 @@ export const testInitUserSolPubkey = async ({
   message,
   ethPrivateKey,
   newUserPublicKey,
-  newUserAcctPDA,
+  userAccountAddress,
```

Comment:
we did PDA => Account elsewhere; i think we should stick to PDA => Account here instead of PDA => AccountAddress - wdyt?